conformance-test/retryStrategy.ts

```diff
@@ -64,20 +64,7 @@ const methodMap: Map<String, String[]> = new Map(
 
 const DURATION_SECONDS = 600; // 10 mins.
 
-const TESTS_PREFIX = `storage-retry-tests-${shortUUID()}-`;
-const FILE_OPTIONS = {
-  preconditionOpts: {
-    ifGenerationMatch: 0,
-    ifMetagenerationMatch: 1,
-  },
-};
-
-const BUCKET_OPTIONS = {
-  preconditionOpts: {
-    ifMetagenerationMatch: 1,
-  },
-};
-
+const TESTS_PREFIX = `storage.retry.tests.${shortUUID()}.`;
```

Comment:
Changed to using `.` due to https://cloud.google.com/storage/docs/naming-buckets (see section about using a period in names).

Reply:
what is the benefit of using `.` over `-`?